### ISSUE TITLE
Readme: How to use pretrained model

### ISSUE THREAD

**Issue by chrisby** [helme/ecg_ptbxl_benchmarking#1]
Hey there!
Thanks for the paper and the repo those are both great resources. I was wondering if you could write a small howto on fine-tuning one of your pre-trained models. That'd be very helpful.

Best,
Christian

**Comment by helme** [comment]
Hi Christian,

in [this notebook](https://github.com/helme/ecg_ptbxl_benchmarking/blob/master/code/Finetuning-Example.ipynb) we provide a basic example of finetuning one of our pretrained models (xresnet) from the experiment on all 71 ECG statments. For demonstration purposes we added a pretrained model and the respective standarizer to our repository (output/exp0/), we hope you like it.

In addition we provide an example of finetuning this model on some custom data (in this case also PTB-XL but with superdiagnostic satements (5 in total). We tried to comment as much as possible, but if you still have questions, feel free to ask ;)

Best,
Patrick
